test/fakes/fake-network-client.ts

```diff
@@ -49,24 +52,33 @@ export class FakeNetworkClientResponses {
    */
   public constructor(
     public readonly getAccountInfoResponse: Response<
-    GetAccountInfoResponse
+      GetAccountInfoResponse
     > = FakeNetworkClientResponses.defaultAccountInfoResponse(),
     public readonly getFeeResponse: Response<
-    GetFeeResponse
+      GetFeeResponse
     > = FakeNetworkClientResponses.defaultFeeResponse(),
     public readonly submitransactionResponse: Response<
-    SubmitTransactionResponse
+      SubmitTransactionResponse
     > = FakeNetworkClientResponses.defaultSubmitTransactionResponse(),
     public readonly getTransactionStatusResponse: Response<
-    GetTxResponse
+      GetTxResponse
     > = FakeNetworkClientResponses.defaultGetTxResponse()
   ) {}
 
   /**
    * Construct a default AccountInfoResponse.
    */
   public static defaultAccountInfoResponse(): GetAccountInfoResponse {
-    return new GetAccountInfoResponse();
+    const balance = new XRPDropsAmount();
+    balance.setDrops(10);
```

Comment:
It doesn't matter that this is a magic number, because we don't assert that we get back `10` in the test, just that a `balance` exists, correct?

Reply:
Yeah, we're just asserting existence. I could name this as a constant, export it and assert that it's exactly equal, but I kinda figured that was a bit over kill since this class is already pretty large and unwieldy.